### PR TITLE
Sass Fathead: Switch to new Template 

### PR DESCRIPTION
**Pull request by VeerpalBrar**
Proposed fix for issue #421. I implemented the new template for programming fatheads.

cc @moollaza

Instant Answer page: https://duck.co/ia/view/sass

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/fathead/sass/parse_functions.py [hunk 1]
```diff
@@ -251,8 +234,8 @@ def parse_for_data(self):
                 if example:
                     abstract =  abstract + example
                 if parameter:
-                     abstract = "%s<br>%s"%(abstract, parameter)
-
+                     abstract = "%s%s"%(abstract, parameter)
+                abstract = '<div class="prog__container">' + abstract + "</div>"
```
Comment (moollaza) [inline lib/fathead/sass/parse_functions.py]: @VeerpalBrar we recently found out that using a <div> here is going to cause unseen problems -- can you switch it to a <section> instead?

I'm going to update the docs shortly!

Thanks 👍

Comment (VeerpalBrar) [inline lib/fathead/sass/parse_functions.py]: I made this change in both files.

**Comment by moollaza** [comment]
@VeerpalBrar Thanks! This LGTM :+1:

This is live on Beta: https://beta.duckduckgo.com/?q=sass+rgb&t=ho&ia=about&iax=1

There's a small CSS bug on Beta though that will resolve itself shortly. Once it's done the subheading will display correctly, like it does here: https://moollaza.duckduckgo.com/?q=sass+rgb&t=ho&ia=about&iax=1

**Pull request merged**